tests/integration/run_tests.py

```diff
@@ -1032,6 +1032,111 @@ async def test_metax_publish_dataset(sess, folder_id):
             metax_res = await metax_resp.json()
             assert metax_res["state"] == "published"
 
+            # this data is synced with /test_files/doi/test_doi.json
+            # if data changes inside the file it must data must be reflected here
+            expected_rd = {
```

Comment:
can this object be loaded from an external file, e.g. we have a file under `test_files/metax/metax.json` ?